vignettes/geologic-map.Rmd

```diff
@@ -88,10 +88,10 @@ head(hc, n = 1)
 ```
 That looks like what we need! Now let's move on to visualizing the data. 
```

Comment:
Why? As above, I think many readers won't understand what they are looking at here (or what they should be noticing about it that is positive). I don't think this is super important, so we could just remove this line viewing the object if it's not necessary.

Reply:
I've added a few lines to show why it matches up, but feel free to remove if you think this is superfluous!